deployments/local-setup-environment/provisioning/install-commons.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing common tools..."
+
+sudo apt-get update
+
+sudo apt-get install -y \
+  curl \
+  wget \
+  git \
+  unzip \
+  jq \
+  python \
+  build-essential
+
+if ! [ -x "$(command -v unzip)" ]; then echo "unzip installation failed"; exit 1; fi
+if ! [ -x "$(command -v jq)" ]; then echo "jq installation failed"; exit 1; fi
```

Comment:
Why do we check only `jq` and `unzip`, and not e.g. `curl` or `wget`? I'd expect `apt-get` to fail if one of those cannot be installed.

Reply:
Yeah, you're right. I've added them later, during the work on Docker builder an didn't put checks for them.